scripts/dataset/main.js

```diff
@@ -1,6 +1,8 @@
-import { release } from './index.js';
+import cron from 'croner';
```

Comment:
I imagine croner will not have the same problems as node-schedule but how can you be sure ?